modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -1,126 +1,132 @@
 
-* Engine server
-* Engine database, including all data on active and archived process instances
-* Organization definition
-* Runtime data in Bonita Home, _until 7.3_
-* Configuration files in Bonita Home, which are replaced with the default configuration files for the new version
-* Log files from the previous version are not impacted by migration
+Furthermore, from the database point of view, as any operations on a production system, an update is not a zero-risk operation. +
+Therefore, it is also strongly recommended to xref:back-up-bonita-bpm-platform.adoc[backup your database] before launching the updating procedure.
+====
 
-The following are not migrated automatically:
+=== JRE requirements
 
-* Configuration of the platform: Before 7.3 in the _Bonita Home_ folder and after 7.3 in database. Reapply your customizations manually after the migration script has finished (using xref:bonita-bpm-platform-setup.adoc#update_platform_conf[platform setup tool] if migrated to 7.3.0+).
-* Deployed process definitions: The processes will continue to run using the definition created in the previous version of Bonita.
-* Process definition sources (`.bos` files): Migrate these by importing them into the new version of Bonita Studio.
-* +++<a id="bdm_redeploy">++++++</a>+++Business data model, and the business data database: if the migration path include version `7.0.0`,`7.2.0` or `7.2.4`, the Business data model must be redeployed after migration. You can pause the tenant before migration, as a tenant admin, so that you'll be able to redeploy the BDM on a paused tenant once migration is done, using  link:define-and-deploy-the-bdm[Define and deploy the BDM]). Otherwise, no action is required.
-* Custom connectors, actor filers, data types: These might continue to work in the new version, but should be tested, depending on your custom code.
-* Custom pages: These might continue to work in the new version, but should be tested depending on your custom code.
-* Custom reports: These might continue to work in the new version, but should be tested depending on your custom code.
-* REST API Extensions: These might continue to work in the new version, but should be tested depending on your custom code.
+* Bonita 7.0 only supports JRE version 7. If updating to Bonita 7.0 to 7.4.x, please update your JRE to version 7.
+* Bonita 7.5 only supports JRE version 8. If updating to Bonita 7.5 to Bonita 2021.1-0811, please update your JRE to version 8.
+* Bonita 2021.2 only supports JRE version 11. If updating to Bonita 2021.2 or greater, please update your JRE to version 11.
 
-When the script has finished,
-you need to complete the migration by unzipping and configuring a bundle for the new version.
-See  <<migrate,Migrate your platform>> for step-by-step instructions.
+For more information, go to the Customer Service Center and read the Support Guide and Supported Environment Matrix for Server.
```

Comment:
this is not friendly for Community users that may not have access to the CSC.
The information are spread between versions in the documentation.

Reply:
removed